scipy/cluster/hierarchy.py

```diff
@@ -1674,15 +1674,15 @@ def _plot_dendrogram(icoords, dcoords, ivl, p, n, mh, orientation,
 
         lbls = ax.get_xticklabels()
         if leaf_rotation:
-            map(lambda lbl: lbl.set_rotation(leaf_rotation), lbls)
+            list(map(lambda lbl: lbl.set_rotation(leaf_rotation), lbls))
```

Comment:
I'm all for it, just tried to keep it as close as possible to the original code. Will change...